src/components/AnchorList/index.js

```diff
@@ -0,0 +1,179 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import Modal from '../Modal'
+import {
+  AncherContainer,
+  AnchorContent,
+  AnchorTitle,
+  AnchorLinksMob,
+  AnchorLinks,
+  AnchorIconLink,
+  AnchorIconTitle,
+  AnchorContainerDes,
+  AnchorLinksContainerDes,
+  AnchorContentDes,
+  AnchorItemsDes,
+  AnchorItemsText,
+} from './style'
+import Icon from '~/components/Icon'
+import { WidthLimiterContainer } from '../Layout/styles'
+import { Tablet, Desktop } from '../Utilities/Media'
+import Text from '../Utilities/Text'
+import Img from 'gatsby-image'
+
+const AnchorList = ({ anchorQuery }) => {
+  const [open, setOpen] = useState(false)
+  const [ministerState, setMinisterState] = useState(0)
+  const [stateQueury, setStateQuery] = useState('')
+
+  const links = []
+
+  anchorQuery?.links
+    ? anchorQuery.links.map((info) => {
+        Object.values(info).map((values) => {
+          links.push(values)
+        })
+      })
+    : ''
```

Comment:
you should store the links insiide a state. Having them outside can cause unexpected behaviours. 

`const [links, setLinks] = useState([])`

`useEffect(() => {
   if (anchorQuery?.links?.length > 0) {
      const links = anchorQuery.links.map(info => Object.values(info))

       setLinks(links)
   }
}, [anchorQuery])`

It may be a bit off